### PR TITLE
Add new IBD Studyviewerconfig fields

### PR DESCRIPTION
**Pull request by nss10**
Link to Jira ticket if there is one: [PXP-8999](https://ctds-planx.atlassian.net/browse/PXP-8999)

### Environments
* IBDGC Commons

### Description of changes
* Added two new fields `data_type` and `estimated_study_completion` in Studyviewer config of IBDGC commons

### CODE REVIEW

File: ibdgc.datacommons.io/portal/gitops.json [hunk 1]
```diff
@@ -408,19 +408,17 @@
       "rowAccessor": "project_id",
       "listItemConfig": {
         "blockFields": ["description"],
```
Comment (paulineribeyre) [inline ibdgc.datacommons.io/portal/gitops.json:410]: i know it's unrelated, but i think we're going to want to change the study viewer `openMode` to `close-all` 🤔

**Review by paulineribeyre — COMMENTED** [review]
lgtm but waiting for #3749

File: ibdgc.datacommons.io/portal/gitops.json (hunk 1 above)
Comment (nss10) [reply ibdgc.datacommons.io/portal/gitops.json:410]: Should I make that change?

Comment (paulineribeyre) [reply ibdgc.datacommons.io/portal/gitops.json:410]: please do 🙏

Comment (nss10) [reply ibdgc.datacommons.io/portal/gitops.json:411]: Done! 👍

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**